backend/package.json

```
@@ -6,7 +6,7 @@
     "scripts": {
         "build": "cross-env NODE_ENV=production run-s clean copy-and-lock compile build-frontend pack",
         "build-frontend": "cd ../frontend && npm run build && cp -R build/* ../backend/build",
-        "copy-and-lock": "chmod +w src/sharedCopy/* && cp -R ../frontend/src/shared/* ./src/sharedCopy && chmod -w src/sharedCopy/*",
+        "copy-and-lock": "chmod +w src/sharedCopy/* && cp -R ../frontend/src/shared/* ./src/sharedCopy && chmod -w src/sharedCopy/* && chmod +w ./src/cli-version.txt && cp ./src/cli-version.txt ./build",
```

```suggestion
        "copy-and-lock": "chmod +w src/sharedCopy/* && cp -R ../frontend/src/shared/* ./src/sharedCopy && chmod -w src/sharedCopy/* && cp ./src/cli-version.txt ./build",
```
Shouldn't need the chmod +w, that's just allowing write access. Doing it for the sharedCopy files so that we lock after copy.

---

Alternatively if you want to just keep the cli-version.txt file in the src directory that's fine too, then you might not have to copy on build. Either way is fine.

---

it's necessary the copy because it'll broke because we are using the folder `build` of the backend